man/tmpfiles.d.xml

```diff
@@ -585,9 +585,31 @@ w- /proc/sys/vm/swappiness - - - - 10</programlisting></para>
       <para>The age of a file system entry is determined from its last
       modification timestamp (mtime), its last access timestamp (atime),
       and (except for directories) its last status change timestamp
-      (ctime). Any of these three (or two) values will prevent cleanup
-      if it is more recent than the current time minus the age
-      field.</para>
+      (ctime). By default, any of these three (or two) values will
+      prevent cleanup if it is more recent than the current time minus
+      the age field. To restrict the deletion based on particular type
+      of file timestamps, the age-by argument can be used.</para>
+
+      <para>The age-by argument, when (optionally) specified along
+      with age will check if the file system entry has aged by the
+      type of file timestamp(s) provided. It can be specified by
+      prefixing the age argument with a set of file timestamp types
+      followed by a colon character <literal>:</literal>, i.e.,
+      <literal><replaceable>age-by</replaceable>:<replaceable>cleanup-age</replaceable></literal>.
+      The argument can be one of (or a set containing):
+      <constant>a</constant>,
+      <constant>b</constant>,
+      <constant>c</constant>, or
+      <constant>m</constant>,
```

Comment:
hm,, this doesn't feel complete to me yet. i.e. for dirs one wants to specify a different combination, i.e. our default combination is abm for dirs and abmc for everything else already. We should be able to express this default behaviour here.

hence, taking inspiration of the cryptic chmod expressions do this, use uppercase for dirs, lowercase for evertyhing else.

i.e. the combination of "abcmABM" would be our default.

Reply:
So, separate sets for files and directories? Cool.